WDAC-Policy-Wizard/app/MSIX/AllowMicrosoft - SingleFormat.xml

```diff
@@ -0,0 +1,171 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why are we keeping single-format policies around? All in-support versions of windows support multi-policy?

Reply:
Not true. Server 2016 and 2019 are in service

Reply:
WS2019 doesn't support Multi-policy?

Reply:
Sadly no. It's based on 1809. Multi policy was introduced in 1903